ui/src/kapacitor/components/handlers/Pagerduty2Handler.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {SFC} from 'react'
+import HandlerInput from 'src/kapacitor/components/HandlerInput'
+import HandlerEmpty from 'src/kapacitor/components/HandlerEmpty'
+
+interface Props {
+  selectedHandler: {
+    enabled: boolean
+  }
+  handleModifyHandler: () => void
+  onGoToConfig: () => void
+  validationError: string
+}
+
+const Pagerduty2Handler: SFC<Props> = ({
+  selectedHandler,
+  handleModifyHandler,
+  onGoToConfig,
+  validationError,
+}) =>
+  selectedHandler.enabled ? (
```

Comment:
Consider changing this to not use a ternary

Reply:
👍